src/EditorFeatures/Core/Wrapping/SeparatedSyntaxList/AbstractSeparatedSyntaxListWrapper.cs

```diff
@@ -5,6 +5,8 @@
 
 namespace Microsoft.CodeAnalysis.Editor.Wrapping.SeparatedSyntaxList
 {
+    using Microsoft.CodeAnalysis.Indentation;
```

Comment:
here.

Reply:
can't.  if we move it, then different symbols will be found in Microsoft.CodeAnalysis.Editor.